lib/raygun.messageBuilder.ts

```diff
@@ -137,7 +137,7 @@ export class RaygunMessageBuilder {
     this._filters = options.filters || [];
 
     this.message = {
-      occurredOn: new Date(),
+      occurredOn: options.timestamp || new Date(),
```

Comment:
Let's hope no one actually meant 0 as a legit timestamp! :-)

Reply:
We checked that, the Raygun server ignores any values of `occurredOn` that are like 128 days apart from the real date.